src/libkomwm.py

```diff
@@ -722,12 +722,8 @@ def rule_sort_key(dict_):
                                 # A casing line explicitly defined via ::object_id.
                                 dr_line.priority = get_drape_priority(cl, 'line', st.get('object-id'))
                                 store_visibility(cl, 'line', st.get('object-id'), zoom)
-                            dd = st.get('casing-dashes', st.get('dashes', []))
-                            for i in dd:
-                                if float(i) < 1:
-                                    print(f'WARNING: forcing minimum dashdot value to 1 in "{dd}" for z{zoom} line(casing) {cl}')
-                                    i = 1.0
-                                dr_line.dashdot.dd.extend([i * WIDTH_SCALE])
+                            for i in st.get('casing-dashes', st.get('dashes', [])):
+                                dr_line.dashdot.dd.extend([float(i)])
```

Comment:
WIDTH_SCALE not needed ?

Reply:
UPD: WIDTH_SCALE == 1. I'd keep it for consistency with other valued.

Reply:
Effectively `WIDTH_SCALE` is not used anywhere, so I see no point in keeping it in the code.

I can remove all usages of it (its just a few) at once in this PR for consistency. WDYT?

Reply:
Remove if you think it will not be used.

Reply:
Done